From ed260e4a9c42b1079ee4046cb3a481bf9c3653c3 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Thu, 8 Aug 2024 08:11:53 +0530
Subject: [PATCH 10/10] Switch to adaptive launcher icon

Adaptive icon is already present inside apk, just use that one.
Required for themed icon.

sed -i "s/ic_launcher_camera_cv/ic_launcher_camera/" AndroidManifest.xml
---
 AndroidManifest.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 7dd77b57..1a16e92e 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -77,7 +77,7 @@
         <package android:name="com.miui.gallery"/>
         <package android:name="com.xiaomi.wearable"/>
     </queries>
-    <application android:allowBackup="false" android:appComponentFactory="androidx.core.app.CoreComponentFactory" android:extractNativeLibs="false" android:hardwareAccelerated="true" android:icon="@mipmap/ic_launcher_camera_cv" android:label="@string/camera_label" android:largeHeap="true" android:name="com.android.camera.CameraAppImpl" android:networkSecurityConfig="@xml/network_security_config" android:resizeableActivity="false" android:supportsRtl="true" android:theme="@style/ThemeCamera">
+    <application android:allowBackup="false" android:appComponentFactory="androidx.core.app.CoreComponentFactory" android:extractNativeLibs="false" android:hardwareAccelerated="true" android:icon="@mipmap/ic_launcher_camera" android:label="@string/camera_label" android:largeHeap="true" android:name="com.android.camera.CameraAppImpl" android:networkSecurityConfig="@xml/network_security_config" android:resizeableActivity="false" android:supportsRtl="true" android:theme="@style/ThemeCamera">
         <meta-data android:name="required_permissions" android:value="@string/required_permissions_supportModifiable"/>
         <meta-data android:name="miui.supportRequiredModifiable" android:value="true"/>
         <meta-data android:name="com.miui.bugreport.LOG_DIR" android:resource="@xml/log_paths"/>
-- 
2.49.0

